administration/src/util/helper.ts

```diff
@@ -10,3 +10,6 @@ export const updateArrayItem = <T>(array: T[], updatedItem: T, index: number): T
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export type UnionToIntersection<U> = (U extends any ? (x: U) => void : never) extends (x: infer I) => void ? I : never
+
+const multipleSpacePattern = /\s\s+/g
+export const removeMultipleSpaces = (value: string): string => value.replace(multipleSpacePattern, ' ')
```

Comment:
Maybe this could be more general and also remove trailing or leading spaces

Reply:
we would need a separate function then since if you trim leading spaces immediately after input no one can type in forename and name

Reply:
Since this function is used on state changes it shouldn't remove these single spaces since it would block the user input
The trimming for single spaces will be handled in the generateCardInfo to not impact on form typing